public/scripts/open-project.js

```diff
@@ -0,0 +1,115 @@
+(function() {
+  require(["jquery"], function($) {
+    var projects = document.querySelectorAll("tr.bramble-user-project");
+
+    function generateUrl(path, date) {
+      var url = window.location;
+      var qs = url.search;
+      if(qs.length < 1) {
+        qs = "?";
+      }
+      qs += "now=" + encodeURIComponent(date);
+
+      return url.protocol + "//" + url.host + "/newProject/" + encodeURIComponent(path) + qs;
```

Comment:
You're doing this kind of thing in a few places, I wonder if you should move this to a `utils` module and reuse?  Also, see comments above about maybe dropping `protocol`.